code/src/Core/Locations/RemoteTemplatesSource.cs

```diff
@@ -63,11 +72,9 @@ private static void DownloadContent(string sourceUrl, string file)
             }
             catch (Exception ex)
             {
-                string msg = "The templates content can't be downloaded.";
-
-                AppHealth.Current.Exception.TrackAsync(ex, msg).FireAndForget();
-
-                throw;
+                string msg = $"Templates content can't be downloaded right now, we will try it later.";
```

Comment:
It is just a log message to indicate that the download didn't worked and will be try later... Suggestions are welcomed.